src/ActionList/List.tsx

```diff
@@ -36,12 +39,19 @@ interface ListPropsBase {
    * - `"full"` - `List` children are flush (vertically and horizontally) with `List` edges
    */
   variant?: 'inset' | 'full'
+
+  // **
+  role?: string
+
+  tabIndex?: number
+
+  ref?: React.RefObject<HTMLDivElement>
```

Comment:
should we be adding these props on like this or are there existing interfaces that add these in?